.github/workflows/linting-and-tests.yml

```diff
@@ -361,7 +364,7 @@ jobs:
             --set oncall.twilio.authToken="${{ secrets.TWILIO_AUTH_TOKEN }}" \
             --set oncall.twilio.phoneNumber="\"${{ secrets.TWILIO_PHONE_NUMBER }}"\" \
             --set oncall.twilio.verifySid="${{ secrets.TWILIO_VERIFY_SID }}" \
-            --set grafana.replicas=3 \
+            --set grafana.replicas=1 \
```

Comment:
Can't reliably use > 1 grafana replica when using SQLite as the grafana database
https://github.com/bitnami/charts/issues/10905